### ISSUE TITLE
Wrong string replacement

### ISSUE THREAD

**Issue by ogmueller** [ask/rabbitmq-munin#17]
Most of the scripts are doing a "grep -v 'done.$' | sed -e 's/[.=-]/_/g' )" for the config, which will replace a queue name like "abc-def" into "abc_def". The problem with this is, that the same string replacement has to be done with the rabbitmqctl output, because otherwise the values cannot be related to the config data. So in case of the consumer script the last line has to be something like:
HOME=$HOME rabbitmqctl list_queues -p $VHOST name consumers| grep -v "^Listing" | grep -v "done.$" | grep -v "..." | sed -e 's/[.=-]/    _/g' | sed "s/[\t]/.value /g"

This issue is in more or less all scripts.

**Comment by sp00x** [comment]
Indeed, had the same issue here now, and had to monkey patch. I guess there isn't any much activity on this project, judging by the commit history...
